.github/scripts/setup_runtime_ubuntu_20.04.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
The CI fails because this file is named setup_runtime_ubuntu_20.04.sh instead of setup_runtime_ubuntu**-**20.04.